packages/odyssey-react/src/components/Banner/Banner.tsx

```diff
@@ -103,8 +104,16 @@ let Banner = forwardRef<HTMLDivElement, BannerProps>((props, ref) => {
           />
         </div>
       )}
-      {content && <p className={styles.content}>{content}</p>}
-      {children && <section className={styles.actions}>{children}</section>}
+      {content && (
+        <span className={styles.content}>
+          <Text as="p" children={content} />
+        </span>
+      )}
+      {children && (
+        <span className={styles.actions}>
```

Comment:
Previous code used `<section>` here and this has switched to `<span>`. Semantics aside, block to inline seems like a big change. Was this intended?

Reply:
It was not. Will fix!